language/extensions/move-table-extension/src/lib.rs

```diff
@@ -40,7 +40,7 @@ use std::{
 /// hash over a transaction hash provided by the environment and a table creation counter
 /// local to the transaction.
 #[derive(Copy, Clone, Debug, PartialOrd, Ord, PartialEq, Eq)]
-pub struct TableHandle(pub u128);
+pub struct TableHandle(pub u128, pub u128);
```

Comment:
Can you also make this `{ pub low: u128, pub high: u128 }`? Otherwise it seems risky

Reply:
Isn't the memory repr just the opposite (high-low because of big endian).

Reply:
Ah I didn't think twice before writing the stuff down. Was just trying to point out that I would love to see them being explicitly named. Please check the ordering before committing this!

Reply:
Yes: naming them would be very useful to prevent developers accidentally flipping them.

Reply:
it's little endian (to comply with bcs)